core/src/main/java/com/amplifyframework/predictions/models/IdentifiedText.java

```diff
@@ -49,13 +49,23 @@ public String getText() {
     }
 
     /**
-     * Gets the page value.
-     * @return the page
+     * Gets the page of the document in which this text was
```

Comment:
Further elaborated. The sneaking suspicion is that this is actually an unnecessary property that was defined simply because Textract provides this value. It doesn't seem to be applicable to other potential types of implementation